library/general/src/modules/Hooks.rb

```diff
@@ -133,11 +133,8 @@ def to_s
       end
 
       def verify!
-        if path.exist?
-          path
-        else
-          raise "Hook search path #{path} does not exists"
-        end
+        return path if path.exist?
+        raise "Hook search path #{path} does not exists"
```

Comment:
in fact I prefer opposite order as usually it is in verify - check, check, check, fine...so if there is more check in verify in your way it will be harder to extent.
so something like
```
raise "Hook search path #{path} does not exists" unless path.exist?

path
```